.gitignore

```diff
@@ -34,12 +34,4 @@
 /upgrade
 /upgrade_errors.txt
 /translations
-/media/images/*
-!/media/images/flags
-!/media/images/apple-touch-icon.png
-!/media/images/avatar.png
-!/media/images/favicon.ico
-!/media/images/mautic_logo*
-!/media/images/.htaccess
-/media/dashboards/*
-!/media/dashboards/*.json
\ No newline at end of file
+/media/*
```

Comment:
Was there a reason for this change? There are some files we want to track and some we don't. This change will ignore all files and folders in /media/ folder.